goal_src/jak3/pc/features/speedruns.gc

```diff
@@ -72,6 +72,76 @@
 (defmethod set-category! ((this speedrun-info) (category speedrun-category))
   (set! (-> this category) category))
 
+(defconstant HERO_MODE_SECRETS
+  (game-secrets hero-mode
+                endless-ammo
+                invulnerable
+                endless-dark
+                endless-light
+                unlimited-turbos
+                vehicle-hit-points
```

Comment:
This is not taken in hero mode runs because you want the car to die fast.
https://youtu.be/AnrHvAayHmE?t=227